libnavigation-core/src/test/java/com/mapbox/navigation/core/telemetry/MapboxNavigationTelemetryTest.kt

```diff
@@ -342,6 +344,80 @@ class MapboxNavigationTelemetryTest {
         assertEquals(10, events.size)
     }
 
+    @Test
+    fun cache_feedback_is_cached() {
+        baseMock()
+        mockAnotherRoute()
+
+        baseInitialization()
+        val cacheFeedbackType = FeedbackEvent.POSITIONING_ISSUE
+        cacheUserFeedback(feedbackType = cacheFeedbackType)
+        val cacheFeedbackDescription = "cacheFeedbackDescription"
+        cacheUserFeedback(description = cacheFeedbackDescription)
+        updateSessionState(FREE_DRIVE)
```

Comment:
not needed

Reply:
if we don't add this line, I think we are not flushing the location buffer which in result will not call the cached user feedback event's `onBufferFull` callback.

Reply:
got it, you are right!